src/splashscreen.ts

```diff
@@ -0,0 +1,46 @@
+const possibleSplashFlavors: string[] = [
+	'Innovative Splash Text!',
+	'Contribute On Github!',
+	'Pretty Slick, NGL.',
+	'Back Again?',
+	'Get That Nuke!',
+	'At Least 10 FPS.',
+	'Slidehopping Into Your DMs.',
+	'Raining On Their Parade.',
+	'Buy Kraxen A Coffee!',
+	'Splish Splash.',
+	'Are We Back?',
+	'Farming Wallbangs.',
+	'Joke About Shotgun Players',
+	'Loading The Worst Person Shooter...',
+	'Are We Loading?',
+	'Pro Tip: WASD To Move.',
+	'Masters Degree In Krunker.',
+	'Deleting System32...',
```

Comment:
nice, but i'd rather not include the deleting system32 because it might make ppl complain in the discord lmao

Reply:
Gotcha, I removed that one and added some more:

- "Going nuclear..."
- "You Should Clanhop!"
- "Lobbyhopping!"
- "Hitting a One-Burst..."
- "Hup!"
- "What mod are you using?"
- "BOOM HEADSHOT!"
- "Everyone runs faster with a knife!"
- "Everyone knows that!"
- "Open Source!"
- "40% Packet Loss"
- "Woah Mama!"
- "Flatline!"
- "RETIRED!"
- "Breakcore is a real genre!"
- "Not the car part."
- "Userscripts are sweet!"
- "Learn Typescript Today!"
- "Use the Custom Matchmaker!"
- "I CAN'T COMPETE!"
- "dude..."
- "Also Try Terraria!"
- "Also Try Minecraft!"
- "Shift Space Shift Space..."
- "Fragging Out!"
- "Intelligent?"

Reply:
haha nice